### PR TITLE
do_Get for all categories

### PR DESCRIPTION
**Pull request by jvanveld97**
1. `git fetch --all`
2. switch to this branch with `git checkout jv-categories`
3. open postman
4. perform a get request to `http://localhost:9999/categories`
5. Verify that you get a list of categories back as JSON with the following properties for each category: 
```
[
    {
        "id": 1,
        "label": "News"
    },
    {
        "id": 2,
        "label": "Technology"
    }
]
```

### CODE REVIEW

**Review by stevebrownlee — CHANGES_REQUESTED** [review]
Please provide a comprehensive PR description of how to test this code. For example,

1. `git fetch --all`
2. Switch to this branch with `git checkout jv-categories`
3. Open Postman
4. Perform a GET request to `http://localhost:8000/categories`
5. Verify that you get a list of categories back as JSON with the following properties for each category.
...list the properties

**Comment by ericlheidel** [comment]
2024-03-05 1:16PM
testing completed functioning

Waiting on Re-Review before merging

**Pull request merged**
